barterdude/hooks/metrics/prometheus.py

```diff
@@ -1,6 +1,38 @@
-from barterdude.hooks import BaseHook
+from aiohttp import web
 
+from barterdude import BarterDude
+from barterdude.hooks import HttpHook
 
-class Prometheus(BaseHook):
-    def __init__(self, gateway_url: str):
-        self.__gateway_url = gateway_url
+try:
+    from prometheus_client import CollectorRegistry, generate_latest
+except ImportError:
+    print("""
+    Please install extra dependency with:
+        `pip install barterdude[prometheus]`
+    """)
```

Comment:
Isso é mesmo necessário? Não deveríamos colocar apenas no requiriments.txt?

Reply:
This is for optional install.... There are other ways to measure metric and Prometheus is just one.... This should be on requirements_dev.txt too